### PR TITLE
test(kubeflow): enable tests & tide for kubeflow/pipelines

### PR DESCRIPTION
**Pull request by Bobgy**
Part of https://github.com/kubernetes/test-infra/issues/14343
This PR should be merged at the same time as https://github.com/kubernetes/test-infra/pull/20553.

/assign @chaodaiG

### CODE REVIEW

**Comment by Bobgy** [comment]
/hold
Let's first review the changes

**Comment by Bobgy** [comment]
How can I fix the dashboard configuration?

File: prow/oss/plugins.yaml [hunk 1]
```diff
@@ -138,6 +138,23 @@ plugins:
   - lifecycle
   - override
 
+  kubeflow:
```
[the remark this answers is not in the document]
Comment (Bobgy) [reply prow/oss/plugins.yaml:141]: It seems to me, these plugins are not risky to migrate all at once. I only want to move tide and trigger at a repo by repo pace.

Does that work?

[the remark this answers is not in the document]
Comment (Bobgy) [reply prow/oss/plugins.yaml:141]: @chaodaiG

**Comment by Bobgy** [comment]
@chaodaiG 
What does the error message mean?

```
2021/01/21 09:45:55 FAIL: could not write config: 1 error occurred:
	* configuration error for (TestGroup) kubeflow-pipelines-periodic-project-cleanup: Each Test Group must be referenced by at least 1 Dashboard Tab.
```

Do I need to set up the new dashboard first?

**Comment by Bobgy** [comment]
I need to add an annotation:
https://github.com/GoogleCloudPlatform/oss-test-infra/blob/39a1aaff56098d71fb4517e3705ff1c59d4a64d4/prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml#L55

**Comment by Bobgy** [comment]
Hmm, I'm getting the same error message again
```
{"component":"unset","file":"prow/cmd/checkconfig/main.go:79","func":"main.reportWarning","level":"warning","msg":"[the following orgs or repos require the lgtm label for merging but do not enable the lgtm plugin: [repo: kubeflow/pipelines], the following orgs or repos require the approved label for merging but do not enable the approve plugin: [repo: kubeflow/pipelines]]","severity":"warning","time":"2021-01-22T06:51:20Z"}
{"component":"unset","file":"prow/cmd/checkconfig/main.go:82","func":"main.reportWarning","level":"fatal","msg":"Strict is set and there were warnings","severity":"fatal","time":"2021-01-22T06:51:20Z"}
```

**Comment by Bobgy** [comment]
Can we disable the strict check for kubeflow org temporarily until it has fully migrated?

**Comment by chaodaiG** [comment]
Instead of turning off `strict`, can we enable the plugins here https://github.com/GoogleCloudPlatform/oss-test-infra/pull/676/files#diff-61d4ecf8ed653ce1a03b8a3758c9fa61af90500fd05f6580dbafcfef3ec780f0R141 instead?

**Comment by Bobgy** [comment]
> Instead of turning off `strict`, can we enable the plugins here https://github.com/GoogleCloudPlatform/oss-test-infra/pull/676/files#diff-61d4ecf8ed653ce1a03b8a3758c9fa61af90500fd05f6580dbafcfef3ec780f0R141 instead?

@chaodaiG makes sense, will two set of plugins working together cause problems?

**Comment by chaodaiG** [comment]
> > Instead of turning off `strict`, can we enable the plugins here https://github.com/GoogleCloudPlatform/oss-test-infra/pull/676/files#diff-61d4ecf8ed653ce1a03b8a3758c9fa61af90500fd05f6580dbafcfef3ec780f0R141 instead?
> 
> @chaodaiG makes sense, will two set of plugins working together cause problems?

We have already seen trigger plugins work side by side and worked just fine. The other plugins imo are less harmful, like lgtm, approve plugin should already be able to handle cases where `lgtm` and `approve` label already exist, so I think we should be fine with two sets of plugins enabled

**Pull request merged**